profile.js

```diff
@@ -14,9 +14,11 @@ async function testRollup() {
 }
 
 function getStats(times) {
-  const mean = times.reduce((acc, time) => acc + time, 0) / times.length;
-  const stdDev = times.length > 1 ? Math.sqrt(times.reduce((acc, time) => acc + Math.pow(time - mean, 2), 0) / (times.length - 1)) : 0;
-  return {mean, stdDev};
+  const mean = times.length > 0 ? times.reduce((acc, time) => acc + time, 0) / times.length : 0;
+  const stdDev = times.length > 0 ? Math.sqrt(times.reduce((acc, time) => acc + Math.pow(time - mean, 2), 0) / (times.length)) : 0;
```

Comment:
I think the `N-1` in the divisor is the correct calculation because we have a sample of builds and not the full population.

https://www.statology.org/population-vs-sample-standard-deviation/

```suggestion
  const mean = times.length > 0 ? times.reduce((acc, time) => acc + time, 0) / times.length : 0;
  const stdDev = times.length > 1 ? Math.sqrt(times.reduce((acc, time) => acc + Math.pow(time - mean, 2), 0) / (times.length - 1)) : 0;
```